From 6264195cfa2b85c34b0d9c3c7db99c00ee5564f0 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 23 May 2017 18:07:15 +0800
Subject: [PATCH 1803/5242] MLK-14966-5 mmc: imx: add ARCH_MXC_ARM64 into
 build dependency

commit  0af0d1bc52ee39710cd9b9e931c1a80bc872bb29 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add ARCH_MXC_ARM64 for i.MX mmc driver to support
ARM64 platforms.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mmc/host/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/Kconfig b/drivers/mmc/host/Kconfig
index e0aca55..8c85364 100644
--- a/drivers/mmc/host/Kconfig
+++ b/drivers/mmc/host/Kconfig
@@ -200,7 +200,7 @@ config MMC_SDHCI_CNS3XXX
 
 config MMC_SDHCI_ESDHC_IMX
 	tristate "SDHCI support for the Freescale eSDHC/uSDHC i.MX controller"
-	depends on ARCH_MXC || COMPILE_TEST
+	depends on ARCH_MXC || ARCH_MXC_ARM64 || COMPILE_TEST
 	depends on MMC_SDHCI_PLTFM
 	select MMC_SDHCI_IO_ACCESSORS
 	help
-- 
1.7.9.5

